.github/Pull_Request_template.md

```diff
@@ -10,6 +10,12 @@ _Benchmarks PR (if needed): https://github.com/IntelPython/scikit-learn_bench/pu
 
 ---
 
+PR should start as a draft, then move to ready for review state after CI passed and checkboxes closed.
```

Comment:
Does "after CI passed" include the `/intelci: run` parts?

Reply:
Since external users are not able to read intelci, no.